shared/src/static/taskIdsToLicenseSources.ts

```diff
@@ -0,0 +1,33 @@
+/*
+TODO: Needs to be confirmed with Content
+
+apply-for-shop-license - Which cosmetology license/licenses?
+public-accountant-license and register-accounting-firm appear to be duplicative tasks searching the same license
+moving-company-license - does this need to support all 4 types?
+*/
+
+import { LicenseName } from "@businessnjgovnavigator/shared/src/license";
+
+export const taskIdToLicenseName = {
```

Comment:
Can we type this as `Record<LicenseName, String>`?

Reply:
Not able to Type this due to circular dependency